src/styles/index.scss

```diff
@@ -2,2 +1,237 @@
+.page {
   margin: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+}
+
+:root {
+  --main-text-color: #00acdc;
+}
+
+.header {
+  margin: 0;
+  text-transform: uppercase;
+  box-sizing: border-box;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding-inline: 50px;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  display: flex;
+}
+
+.logo {
+  display: flex;
+}
+
+.nav__list {
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.nav__link {
+  color: #060b35;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  margin-left: 20px;
+  text-decoration: none;
+  display: flex;
+
+  &:hover {
+    color: var(--main-text-color);
+  }
+}
+
+.is-active {
+  color: var(--main-text-color);
+  align-items: center;
+  height: 60px;
+  margin-left: auto;
```

Comment:
The `margin-left: auto;` property for the .is-active class seems unnecessary because it does not have any effect on the positioning of the element.